camp/nfutsum/nfutsum-miniproject/XBox Controller Left and Right bumper/src/main/java/frc/robot/commands/Forward.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.Motor;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/** An example command that uses an example subsystem. */
+public class Forward extends CommandBase {
+  private final Motor m_subsystem;
```

Comment:
Member 'm_' naming comes from Hungarian naming (https://en.wikipedia.org/wiki/Hungarian_notation).  I'd characterize it as more of a 1990's and 2000's thing.  Naming like this has fallen out of favor.  Most java development uses camel casing now (https://en.wikipedia.org/wiki/Camel_case).  Perhaps the oracle naming conventions would be ideal for us to follow ... https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html.

Reply:
changed it